dynamic-simulation/dynamic-simulation-api/src/main/java/com/powsybl/dynamic/simulation/DynamicSimulationParameters.java

```diff
@@ -0,0 +1,126 @@
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+package com.powsybl.dynamic.simulation;
+
+import java.util.Map;
+import java.util.Objects;
+
+import com.google.common.base.Supplier;
+import com.google.common.base.Suppliers;
+import com.google.common.collect.ImmutableMap;
+import com.powsybl.commons.config.PlatformConfig;
+import com.powsybl.commons.extensions.AbstractExtendable;
+import com.powsybl.commons.extensions.Extension;
+import com.powsybl.commons.extensions.ExtensionConfigLoader;
+import com.powsybl.commons.extensions.ExtensionProviders;
+
+/**
+ * @author Marcos de Miguel <demiguelm at aia.es>
+ */
+public class DynamicSimulationParameters extends AbstractExtendable<DynamicSimulationParameters> {
+
+    /**
+     * A configuration loader interface for the DynamicSimulationParameters extensions loaded from the platform configuration
+     * @param <E> The extension class
+     */
+    public static interface ConfigLoader<E extends Extension<DynamicSimulationParameters>>
+        extends ExtensionConfigLoader<DynamicSimulationParameters, E> {
+    }
+
+    public static final String VERSION = "1.0";
+
+    public static final int DEFAULT_START_TIME = 0;
+    public static final int DEFAULT_STOP_TIME = 1;
+
+    private static final Supplier<ExtensionProviders<ConfigLoader>> SUPPLIER = Suppliers
+        .memoize(() -> ExtensionProviders.createProvider(ConfigLoader.class, "dynamic-simulation-parameters"));
+
+    public static DynamicSimulationParameters load() {
+        return load(PlatformConfig.defaultConfig());
+    }
+
+    public static DynamicSimulationParameters load(PlatformConfig platformConfig) {
+        DynamicSimulationParameters parameters = new DynamicSimulationParameters();
+        load(parameters, platformConfig);
+        parameters.loadExtensions(platformConfig);
+
+        return parameters;
+    }
+
+    protected static void load(DynamicSimulationParameters parameters) {
+        load(parameters, PlatformConfig.defaultConfig());
+    }
+
+    protected static void load(DynamicSimulationParameters parameters, PlatformConfig platformConfig) {
+        Objects.requireNonNull(parameters);
+        Objects.requireNonNull(platformConfig);
+
+        platformConfig.getOptionalModuleConfig("dynamic-simulation-default-parameters")
+            .ifPresent(config -> {
+                parameters.setStartTime(config.getIntProperty("startTime", DEFAULT_START_TIME));
+                parameters.setStopTime(config.getIntProperty("stopTime", DEFAULT_STOP_TIME));
+            });
+    }
+
+    private int startTime;
+
+    private int stopTime;
+
+    public DynamicSimulationParameters(int startTime, int stopTime) {
+        this.startTime = startTime;
+        this.stopTime = stopTime;
```

Comment:
Assert that startTime and stopTime are positive, and stopTime is after startTime. Add information somewhere (javadoc) about the unit of this parameters.